src/app/home/home.component.html

```diff
@@ -516,7 +516,7 @@
                 processedText?.textMarkings === undefined ||
                 processedText?.textMarkings?.length === 0
                     ? ''
-                    : 'mt-2'
```

Comment:
Nice! This should hopefully fix it.
Btw, now after removing mt-2, we're left with this code
```
{{
                processedText?.textMarkings === undefined ||
                processedText?.textMarkings?.length === 0
                    ? ''
                    : ''
            }}
```
Perhaps we can remove this? Does it do anything now?

Reply:
Search for, interpolation, ternary operator.

Reply:
Once you're done with making changes, please post a small recording of it working, if you can.